addons/web/static/src/views/fields/copy_clipboard/copy_clipboard_field.xml

```diff
@@ -4,12 +4,12 @@
     <t t-name="web.CopyClipboardField" owl="1">
         <div class="d-grid rounded-2 overflow-hidden">
             <Field t-props="fieldProps"/>
-            <CopyButton className="copyButtonClassName" content="props.value" copyText="copyText" successText="successText"/>
+            <CopyButton className="copyButtonClassName" content="props.value" copyText="copyText" disabled="props.disabled" successText="successText"/>
```

Comment:
I'm not sure it's a good idea to support the disabled props for this case, since the text is also displayed next to the button. You could just copy paste the text directly.
I'd support disabled only for the CopyClipBoardButton case. What do you think?

Reply:
Yep, I also think only the button should have this functionality